src/test/resources/api/azure/users/_get/rs.json

```diff
@@ -0,0 +1,52 @@
+[
+  {
+    "ID": 1,
+    "UserName": "User 1",
```

Comment:
you'll never know how much users will you have. so let's just validate this test via object mapper (at least one record in list)

Reply:
how to validate? just list.length? or that at least 1 item from list can be procceded by object mapper